### ISSUE TITLE
False positive: incorrect end of line character error for character that is not at the end of the line

### ISSUE THREAD

**Issue by sblask** [editorconfig-checker/editorconfig-checker#179]
I get `Not all lines have the correct end of line character` in a file containing:

```
command! WindowsToLinux :%s/^M$//
```

the ^M being the actual Windows line break. If I remove the line from my file the error disappears. This is me .editorconfig:

```
root = true

[*]
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true

[*.js]
indent_size = 2

[*.less]
indent_size = 2

```

**Comment by mstruebing** [comment]
Hey @sblask thank you very much for that issue.

Could you upload the affected file here to reproduce it with exactly that file?

**Comment by sblask** [comment]
Silly me, why didn't I think of that, it's in my dotfiles: https://github.com/sblask/dotfiles/blob/23dcf31b6d79e1dd320bc4f4a2ad9985d76405ac/dotfiles_shared/.vimrc.d.symlink/commands.vim#L30

My editorconfig: https://github.com/sblask/dotfiles/blob/master/dotfiles_shared/.editorconfig.symlink

**Comment by sanmai-NL** [comment]
@sblask Is your problem solved? If so, can you close the issue?

**Comment by sblask** [comment]
@sanmai-NL why would it, there was no release or even commit since I opened the issue?

**Comment by sanmai-NL** [comment]
What's silly about your problem?
